README.md

```diff
@@ -52,9 +52,16 @@ go install github.com/u-root/u-root
 **Note: The `u-root` command will end up in `$GOPATH/bin/u-root`, so you may
 need to add `$GOPATH/bin` to your `$PATH`.**
 
+## Templates
+
+To quickly specify a set of commands from u-root, you can use any of the
+templates as defined in [templates.go](templates.go).
+
+## Examples
+
 You can now use the u-root command to build an initramfs. Here are some examples
```

Comment:
let's get rid of the word 'now', it made sense about 9 years ago but :-)

Reply:
Yea, I'll skim through the entire README again and see what else could be reworded - the next part here is a lengthy list of examples and loose explanations. I see some potential for reordering to give them some more structure. More PRs to come!